main.go

```diff
@@ -93,9 +93,16 @@ func main() {
 		os.Exit(1)
 	}
 
+	ns, err := controllers.GetNamespace()
+	if err != nil {
+		setupLog.Error(err, "unable to operator's namespace")
+		os.Exit(1)
```

Comment:
Please don't make this a final error: this feature is mainly for testing so I would prefer to log (Error is fine or Warn as you prefer) with proper message but default on the native scenario (aka hardcoded image)
Wdyt?

Reply:
Changed in latest commit